gateway/api/services/storage/logs_storage.py

```diff
@@ -90,3 +95,40 @@ def get(self, job_id: str) -> Optional[str]:
                 str(e),
             )
             return None
+
+    def put(self, job: Job, log: str) -> None:
+        """
+        Write a log in a file for the given job id
+
+        Args:
+            job (Job): the job to write the logs
+            log (str): the job to write the logs
+
+        Returns:
+            None
+        """
+        log_path = self._get_logs_path(job.id)
+        max_mb = int(settings.FUNCTIONS_LOGS_SIZE_LIMIT)
+        file_size = 1024 * 1024 * 1
+
+        try:
+            file_logger.remove()
+            file_logger.add(
+                log_path,
+                rotation=file_size,
+                retention=math.trunc(max_mb / file_size),
+                compression=None,  # needed in mounted volumes
+                enqueue=True,  # needed in mounted for S3 Fuse
+                format="({level})[{time}] {message}",
+            )
+            file_logger.info(log)
```

Comment:
It would be important to accept as input the level of log to be able to log `info`, `warning` and `error` as minimum.